charts/kyuubi/Chart.yaml

```diff
@@ -20,7 +20,7 @@ name: kyuubi
 description: A Helm chart for Kyuubi server
 type: application
 version: 0.1.0
```

Comment:
question: is there any suggestion/guidance for this version?

Reply:
Nothing in particular. Chart version needs to be changed on each chart release. By release I mean [creating chart archive](https://helm.sh/docs/helm/helm_package/#helm-package) and adding it to [chart repository](https://helm.sh/docs/topics/chart_repository/).
I would prefer to keep `0.1.0` for the initial dev release, which is much closer now because of the official Kyuubi docker image. Not sure how often the chart should be released. For example MinIO, Argo and many others release it quite frequently (for one or few changes), but Airflow releases helm chart only once for each minor (x.Y.z) application version.

Reply:
Educated, thanks @dnskr